lib/modules/tooltip.js

```diff
@@ -244,7 +245,7 @@ export class _Tooltip extends React.Component {
       return null;
     }
   }
-};
+}
```

Comment:
Actually, it looks like the semi is the odd one out here so I think it's not a big deal.